docker/Dockerfile

```
@@ -1,22 +1,36 @@
 FROM ubuntu:22.04
 
-RUN apt update \
-    && apt-get install -y \
+# install dependencies 
+
+RUN apt update
+RUN apt-get install -y \
```

```suggestion
RUN apt-get update && apt-get install -y \
```

apt -> apt-get fixes `#0 0.049 WARNING: apt does not have a stable CLI interface. Use with caution in scripts.`